### DISCUSSION TITLE
Osu!Lazer doesn't open window

### DISCUSSION THREAD

**Discussion by EgirlLani** [ppy/osu#13880] [Q&A]
This has been a problem for a while, and I enjoy Lazer over stable because of the mania keystrokes. It's been happening for over 3 months now, with me trying to open lazer and nothing happening. I assume it was because one time it was bugging, and I continued to restart it, and then it wouldn't open from then on. I'm currently on the latest version of Windows 10, 64 bit using an Alienware m17 if that helps at all, I'd just like to be able to use lazer again

**Comment by frenzibyte** [comment]
Please follow the bug issue template and attach everything mentioned there here so we can help you.

Specifically, attach the logs.
